src/components/TimelineScrubber/TimelineScrubber.js

```diff
@@ -15,8 +15,8 @@ class TimelineScrubber extends Component {
     runTime: PropTypes.number.isRequired,
     /** the current zoom level */
     zoom: PropTypes.number.isRequired,
-    /** Map of points @todo custom validator */
-    points: PropTypes.object.isRequired,
+    // /** Map of points @todo custom validator */
+    // points: PropTypes.object.isRequired,
```

Comment:
Can delete instead of commenting out, still in git

Reply:
https://github.com/digirati-co-uk/timeliner/pull/105/commits/2a2dd1c1bc8862e28c2b0c939e0b25232cebcf5b